src/languageSupport/languages/flux/lsp/monaco.flux.lsp.ts

```diff
@@ -52,6 +55,17 @@ function createLanguageClient(
 
 let worker: Worker, messageReader, messageWriter, prelude
 
+const handleConnectionClose = () => {
+  reportErrorThroughHoneyBadger(new Error('LSP connection closed.'), {
+    name: 'LSP worker',
+  })
+}
+
+const handleConnectionError = ([e, ,]: [Error, unknown, number]) => {
```

Comment:
Hm, I'm not a huge fan of `e` as parameter name. I realize this mostly just passes the value through, but I don't see a harm in using a more descriptive name like, say, `error`. Thoughts?

Reply:
Good idea. Changed, pushed. And I'll adopt this verbosity in the future.